tests/Nox.Types.Tests/NoxCountryNumberTest.cs

```diff
@@ -0,0 +1,309 @@
+﻿namespace Nox.Types.Tests;
+
+public class NoxCountryNumberTest
+{
+    [Theory]
```

Comment:
the inline data is so huge, we could split it to a ClassData, and keep it in its own file, apart from the test class. see https://hamidmosalla.com/2017/02/25/xunit-theory-working-with-inlinedata-memberdata-classdata/